frontend/src/main/pages/HomePage.js

```diff
@@ -25,6 +26,21 @@ export default function HomePage() {
         url: "/api/commons/all"
       }
     );
+  const onSuccess = (commons) => {
+    // Stryker disable next-line all : hard to get variable
+    var existed = new Boolean(false);
+    for(let i = 0; i < commonsJoined.length; i++ ){
+      if(commonsJoined[i].id == commons.id){
+        existed = true;
+      }
+    }
+    if(existed == true){
+      // Stryker disable next-line all : hard to get variable
```

Comment:
Could you clarify why you have to exclude this line? It should be possible to write a test that covers the toast (and it looks like you're doing that in the else case).

Reply:
Oh I forgot to delete this line. I've figured out how to test that toast message finally. Actually, in this project, "// Stryker disable next-line all" does not work. I don't know why. Deleted that line now.